recipes/bob.ap/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "2.0.4" %}
+
+package:
+  name: bob.ap
+  version: {{ version }}
+
+source:
+  fn: bob.ap-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.ap/bob.ap-{{ version }}.zip
+  md5: 49f89a84c3c2cb20aa0ffa49d39415a7
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - bob.extension
+    - bob.blitz
+    - bob.core
+    - bob.sp
+    - scipy
+    - cmake
+    - pkg-config
+    - gcc     # [linux]
```

Comment:
In this package, we're including a header from bob.sp which declares `fftpack.h`, the C interface for a FORTRAN subroutine. We're linking against the fortran binaries, but using a C API for that.